index.js

```diff
@@ -37,8 +38,20 @@ try {
       // nothing to do, break out
       core.info(`Trello card ${trelloCardUrl} already has GitHub link ${githubUrl} attached`)
     } else {
+      let params = [
+        `url=${githubUrl}`
+      ]
+
+      if (attachmentName != null) {
+        // Name parameter can be maximum 256 characters, so trim it to that
+        if (attachmentName.len() > 256) {
+          core.info("'attachment-name' property can be a maximum of 256 characters. Truncating.")
+        }
+        params.push(`name=${attachmentName.substring(0, 257)}`)
+      }
```

Comment:
We might need to worry about things like URL encoding here. Maybe we can use the [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interface to handle that for us?

Reply:
Good idea. Will refactor to pass params separately and use the native tools for it

```
function trelloApiUrl(url, params)
trelloApiUrl("/cards/foo/attchments/", {"a": "foo", "b": "bar"}
```